src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/BehandlingsoversiktService.kt

```diff
@@ -67,7 +68,7 @@ class BehandlingsoversiktService(
                     vedtaksdato = it.vedtakstidspunkt,
                     henlagtÅrsak = it.henlagtÅrsak,
                     revurderFra = it.revurderFra,
-                    vedtaksperiode = vedtaksperioder[it.id],
+                    vedtaksperiode = if (it.resultat != BehandlingResultat.HENLAGT) vedtaksperioder[it.id] else null,
```

Comment:
Er et alternativ til dette å ikke hente vedtaksperioder for de behandlinger som er henlagte?